x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/AutoFollowCoordinator.java

```diff
@@ -272,10 +268,15 @@ void autoFollowIndices() {
             this.autoFollowPatternsCountDown = new CountDown(patterns.size());
             this.autoFollowResults = new AtomicArray<>(patterns.size());
 
-            getLeaderClusterState(remoteCluster, (leaderClusterState, e) -> {
-                if (leaderClusterState != null) {
+            getRemoteClusterState(remoteCluster, metadataVersion + 1, (leaderClusterStateRsp, e) -> {
```

Comment:
`remote` instead of `leader`.

Reply:
And of course same comment to spell out.

Reply:
#36408 already changes this method, so when I merge in master after that pr is merged it should be good.